pkg/sysinfo/hugepages.go

```diff
@@ -46,13 +46,13 @@ func GetHugepages(hnd Handle) ([]Hugepages, error) {
 		if entry.IsDir() && strings.HasPrefix(entryName, "node") {
 			nodeID, err := strconv.Atoi(entryName[4:])
 			if err != nil {
-				klog.Warningf("cannot detect the node ID for %q", entryName)
-				continue
+				klog.Errorf("cannot detect the node ID for %q", entryName)
```

Comment:
Is this change going to break the existing functionality?

Reply:
It should not, because the structure should be stable (and the code should handle it). If it does, better than silently swallowing.